package.json

```diff
@@ -40,5 +40,7 @@
   "gitHooks": {
     "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD"
   },
-  "dependencies": {}
+  "resolutions": {
```

Comment:
That's pretty weird, why are we bound to specific version?

Reply:
Because there is types issues between two diferent versions. We're not able to use createVuesticPlugin from packages/ui (where is vue resolved as 3.2.37) in project like packages/sandbox or pacakges/nuxt where is vue resolved as 3.2.38 - it will throw type mismatch error.